### PR TITLE
Remove exchange checkbox, reword other checkbox, fix some linting (fixes [APP-771])

### PR DESCRIPTION
**Pull request by BrodyHughes**
Fixes APP-####

## What changed (plus any additional context for devs)
- removed network checkbox
- reworded the other checkbox from `"The person I’m sending to has a wallet that supports %{networkName}",` to `"The address I’m sending to supports %{networkName}",`
- fixed some linting errors
- unsure if i need to make more changes to other lang docs

## What to test
- sending on L2s works fine
- checkboxes go away if sends to a certain address on that network is > 3

### CODE REVIEW

File: src/screens/SendConfirmationSheet.tsx [hunk 1]
```diff
@@ -283,9 +276,9 @@ export default function SendConfirmationSheet() {
       let sendsCurrentNetwork = 0;
       transactions.forEach(tx => {
         if (tx.to?.toLowerCase() === toAddress?.toLowerCase()) {
-          sends++;
+          sends += 1;
```
Comment (skylarbarrera) [inline src/screens/SendConfirmationSheet.tsx:279]: curious was this TS?

File: src/screens/SendConfirmationSheet.tsx [hunk 2]
```diff
@@ -526,6 +525,15 @@ export default function SendConfirmationSheet() {
     shouldShowChecks,
   });
 
+  let message;
```
Comment (skylarbarrera) [inline src/screens/SendConfirmationSheet.tsx:528]: very nit but I prefer these in a lil fn so its incapsulated 

getMessage = () {
..blah blah
return message
}

**Review by skylarbarrera — APPROVED** [review]
🔥 🔥 🔥 

PoW: https://cloud.skylarbarrera.com/Screen-Shot-2023-09-21-11-19-26.81.png

File: src/screens/SendConfirmationSheet.tsx (hunk 1 above)
Comment (BrodyHughes) [reply src/screens/SendConfirmationSheet.tsx:279]: It was linter

**Pull request merged**